src/constants.ts

```diff
@@ -190,3 +191,200 @@ export const SCOPES_BY_TITLE = keyBy(
+  },
+  [TranscendPullResource.ProcessingPurposes]: {
+    description: 'The processing purposes in the data inventory.',
+    markdownLink:
+      '[Data Inventory -> Processing Purposes](https://app.transcend.io/data-map/data-inventory/purposes)',
+  },
+  [TranscendPullResource.ActionItems]: {
+    description: 'Onboarding related action items',
+    markdownLink: '[Action Items](https://app.transcend.io/action-items/all)',
+  },
+  [TranscendPullResource.ActionItemCollections]: {
+    description: 'Onboarding related action item group names',
+    markdownLink: '[Action Items](https://app.transcend.io/action-items/all)',
+  },
+  [TranscendPullResource.Teams]: {
+    description: 'Team definitions of users and scope groupings',
+    markdownLink:
+      '[Administration -> Teams](https://app.transcend.io/admin/teams)',
+  },
+  [TranscendPullResource.PrivacyCenters]: {
+    description: 'The privacy center configurations.',
+    markdownLink:
+      '[Privacy Center](https://app.transcend.io/privacy-center/general-settings)',
+  },
+  [TranscendPullResource.Policies]: {
+    description: 'The privacy center policies.',
+    markdownLink:
+      '[Privacy Center -> Policies](https://app.transcend.io/privacy-center/policies)',
+  },
+  [TranscendPullResource.Messages]: {
+    description:
+      'Message definitions used across consent, privacy center, email templates and more.',
+    markdownLink:
+      '[Privacy Center -> Messages](https://app.transcend.io/privacy-center/messages-internationalization), ' +
+      '[Consent Management -> Display Settings -> Messages]' +
+      '(https://app.transcend.io/consent-manager/display-settings/messages)',
+  },
+  [TranscendPullResource.Assessments]: {
+    description: 'Assessment responses.',
+    markdownLink:
+      '[Assessments -> Assessments](https://app.transcend.io/assessments/groups)',
+  },
+  [TranscendPullResource.AssessmentTemplates]: {
+    description: 'Assessment template configurations.',
+    markdownLink:
+      '[Assessment -> Templates](https://app.transcend.io/assessments/form-templates)',
+  },
+  [TranscendPullResource.Purposes]: {
+    description: 'Consent purposes and related preference management topics.',
+    markdownLink:
+      '[Consent Management -> Regional Experiences -> Purposes]' +
+      '(https://app.transcend.io/consent-manager/regional-experiences/purposes)',
+  },
+};
+
+export const SCOPE_DESCRIPTIONS = (scopeMap: {
+  [k in TranscendPullResource]: ScopeName[];
+}): string =>
+  `| Resource | Description | Scopes | Link |\n| --- | --- | --- | --- |\n${Object.entries(
+    RESOURCE_DOCUMENTATION,
+  )
+    .map(
+      ([resource, { description, markdownLink }]) =>
+        `| ${resource} | ${description} | ${scopeMap[
+          resource as TranscendPullResource
+        ]
+          .map((scopeName) => TRANSCEND_SCOPES[scopeName].title)
+          .join(', ')} | ${markdownLink} |`,
+    )
+    .join('\n')}`;
```

Comment:
per this [comment](https://github.com/transcend-io/cli/pull/446/files#r2277952600), we should add "Key name in `transcend.yml`" column:

```suggestion
  `| Resource | Key name in \`transcend.yml\` | Description | Scopes | Link |
| --- | --- | --- | --- | --- |\n${Object.entries(RESOURCE_DOCUMENTATION)
    .map(
      ([resource, { description, markdownLink }]) =>
        `| \`${resource}\` | \`${
          TR_YML_RESOURCE_TO_FIELD_NAME[resource as TranscendPullResource]
        }\` | ${description} | ${scopeMap[resource as TranscendPullResource]
          .map((scopeName) => TRANSCEND_SCOPES[scopeName].title)
          .join(', ')} | ${markdownLink} |`,
    )
    .join('\n')}`;
```